src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -462,8 +462,8 @@ namespace winrt::TerminalApp::implementation
             const auto& profile = _settings->GetProfiles()[profileIndex];
             auto profileMenuItem = WUX::Controls::MenuFlyoutItem{};
 
-            // add the keyboard shortcuts for the first 9 profiles
-            if (profileIndex < 9)
+            // add the keyboard shortcuts based on the number of profiles defined
+            if (profileIndex < profileCount)
```

Comment:
```suggestion
```

Since we're in a loop from 0 to profileCount, we're just guaranteed that this is true. We can simplify this code by removing the check. If you want, you can keep the `{}` for this block to keep it logically separated from the rest of the code, however.

Reply:
Ok sure, make sense. I've removed that condition in the loop and updated the indentation.